base/commands/sql/sql_test.go

```diff
@@ -0,0 +1,113 @@
+package sql_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	_ "github.com/hazelcast/hazelcast-commandline-client/base/commands"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it"
+)
+
+func TestSQL(t *testing.T) {
+	testCases := []struct {
+		name string
+		f    func(t *testing.T)
+	}{
+		{name: "SQL_Interactive", f: sql_InteractiveTest},
+		{name: "SQL_NonInteractive", f: sql_NonInteractiveTest},
+		{name: "SQLOutput_NonInteractive", f: sqlOutput_NonInteractiveTest},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, tc.f)
+	}
+}
+
+func sql_NonInteractiveTest(t *testing.T) {
```

Comment:
I think test names are not appropriate, we should rename them to `mapping_NonInteractive` because in future we will add many more SQL tests.